misc/buildtestdata.sh

```diff
@@ -49,4 +49,4 @@ mkdir -p "${E2EDATA}"
 RSTAMP=$(TZ=UTC python -c 'import time; print("{:08x}".format(0xffffffff - int(time.time() - time.mktime((2020,1,1,0,0,0,-1,-1,-1)))))')
 
 echo "COPY AND PASTE THIS TO UPLOAD:"
```

Comment:
```suggestion
```
Looks like the "COPY AND PASTE" suggestion is no longer applicable since the command is running directly in the script.